posthog/temporal/workflows/bigquery_batch_export.py

```diff
@@ -253,12 +253,21 @@ async def run(self, inputs: BigQueryBatchExportInputs):
                 ),
             )
 
+        except exceptions.ActivityError as e:
+            if isinstance(e.cause, exceptions.CancelledError):
+                workflow.logger.exception("BigQuery BatchExport was cancelled.")
+                update_inputs.status = "Cancelled"
+            else:
+                workflow.logger.exception("BigQuery BatchExport failed.", exc_info=e)
+                update_inputs.status = "Failed"
+
+            update_inputs.latest_error = str(e.cause)
```

Comment:
I'm not sure about having this, rather than it adding a log entry later when we get to it and in the mean while we can look at the workflow.logger exceptions?